content/influxdb/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -66,34 +66,3 @@ the DBRP mapping service checks for the specified database and retention policy
 
```

Comment:
On this page, we'll want to drop line 30:
- [Manually create and manage DBRP mappings](#manually-create-and-manage-dbrp-mappings)